apps/events-helsinki/.testcaferc.json

```diff
@@ -1,6 +1,9 @@
 {
   "clientScripts": [
-    { "module": "@testing-library/dom/dist/@testing-library/dom.umd.js" }
+    { "module": "@testing-library/dom/dist/@testing-library/dom.umd.js" },
+    {
+      "content": "document.cookie='city-of-helsinki-cookie-consents=%7B%22wordpress%22%3Atrue%2C%22linkedevents%22%3Atrue%2C%22matomo%22%3Atrue%2C%22i18next%22%3Atrue%2C%22city-of-helsinki-cookie-consents%22%3Atrue%2C%22servicemap_analytics%22%3Atrue%2C%22servicemap_session%22%3Atrue%2C%22servicemap_ga%22%3Atrue%7D; path=/;'"
```

Comment:
The value is basically json object which is stringified and then url encoded. This could be more readable if the file would be renamed to a javascript file (testcaferc.js) and then value could be evaluated on the fly:
```
document.cookie='city-of-helsinki-cookie-consents=${{encodeURIComponent(JSON.stringify({"wordpress":true,"linkedevents":true,"matomo":true,"i18next":true,"city-of-helsinki-cookie-consents":true,"servicemap_analytics":true,"servicemap_session":true,"servicemap_ga":true}));
```

Reply:
This is not necessaerily need to be done, but just would make the code more readable